src/test/kotlin/com/metalbear/mirrord/utils/DialogFixture.kt

```diff
@@ -0,0 +1,34 @@
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
```

Comment:
Why is this file necessary?

Reply:
represents a dialog like mirrord dialog asking to choose pod